config/publick8s_get-jenkins-io-mirrorbits.yaml

```diff
@@ -88,15 +88,15 @@ ingress:
   hosts:
     - host: get.jenkins.io
       paths:
-        - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$  # Requires the regexp engine of Nginx to be enabled
+        - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html|asc|key)$  # Requires the regexp engine of Nginx to be enabled
```

Comment:
should key be served from mirrors? aren't they untrusted but rely on checking the signature, if mirrors serve the key they can sign the package themselves?